napari/_qt/qt_main_window.py

```diff
@@ -47,14 +47,14 @@ class _QtMainWindow(QMainWindow):
     # to their desired window icon
     _window_icon = NAPARI_ICON_PATH
 
-    def __init__(self, qt_viewer: QtViewer = None, parent=None) -> None:
+    def __init__(self, parent=None) -> None:
```

Comment:
Very nice!

Reply:
thanks nick.  I might do an intermediate PR before this one to unscrew up what I did in the last one.  will ping you when ready

Reply:
ok, #2220 fixed the real issue, this is now just a tiny little PR with a couple lines.  will merge